### ISSUE TITLE
All remote calendars have been fetched.

### ISSUE THREAD

**Issue by valiantgenomics** [obsidianmd/obsidian-sample-plugin#53]
Just a quick issue that's been happening after the latest update.

Every time I open the calendar, or even if I simply move from another tab to the calendar tab.

I get an "**All remote calendars have been fetched.**" notification in the top right corner.

Initially I didn't mind it, but it's happening so frequently that it makes me wonder what's causing it.

I have only 1 calendar., so not sure what the "remote" part means.

I definitely didn't have it in prior releases, so it's due to the latest update.

I would really appreciate it if someone sheds some light on why this is happening.

Thanks a lot!

**Comment by yajamon** [comment]
Hi @valiantgenomics .
The issue you're experiencing may be related to the community plugin you're using. First, try identifying the name of the plugin and contacting its developer.

**Comment by valiantgenomics** [comment]
> Hi @valiantgenomics . The issue you're experiencing may be related to the community plugin you're using. First, try identifying the name of the plugin and contacting its developer.

Hey, sorry about that. I just now noticed, that I've posted this issue in the general Obsidian git by mistake, instead of the plugin-specific one.

I honestly have no idea how this happened, but totally my fault for not paying attention.

I'll move the issue to the correct git now.

The admin/mods can delete this entire thread, as it's obsolete.

Thank you for understanding.
